Model/RatepayInvoicePaymentMethod.php

```diff
@@ -461,4 +473,133 @@ private function _areAddressesAllowed($sBillingCountryId, $sShippingCountryId)
             $sBillingCountryId === $sShippingCountryId
         );
     }
+
+    /**
+     *
+     * @inheritdoc
+     *
+     * @param string                           $sAction
+     * @param \Wirecard\Oxid\Model\Transaction $oParentTransaction
+     * @param array|null                       $aOrderItems
+     *
+     * @return Transaction
+     *
+     * @since 1.2.0
+     */
+    public function getPostProcessingTransaction($sAction, $oParentTransaction, $aOrderItems = null)
+    {
+        $oTransaction = new RatepayInvoiceTransaction();
+        $aBasketResult = $this->_createPostProcessingBasket($oParentTransaction, $aOrderItems);
+
+        /**
+         * @var $oBasket Basket
+         */
+        $oBasket = $aBasketResult[0];
```

Comment:
Its hard to knows what is $aBasketResult[0] and what is $aBasketResult[1] if you did't implement this.

Reply:
there is no context why the total amount of the transaction is now in the method where a basket entity is created for the transaction.

Reply:
restructured it.